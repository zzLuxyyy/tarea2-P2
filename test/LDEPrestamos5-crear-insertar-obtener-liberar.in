crearLDEPrestamos

obtenerPrimeroLDEPrestamos
obtenerUltimoLDEPrestamos

# Creo e inserto el Prestamo 1
crearSocio 12345678 Jim Dear 10 03 2004
crearFecha 01/01/1966
crearLibro 411222335 "Todos los fuegos el fuego" "Julio" "Cortázar" "Ocho cuentos que combinan lo cotidiano con lo fantástico" 1
crearFecha 08/09/2024
crearPrestamo
insertarLDEPrestamos

obtenerUltimoLDEPrestamos

# Creo e inserto el Prestamo 2
crearSocio 23456789 Lucia Perez 10 03 2010
crearFecha 01/01/1961
crearLibro 421233446 "La ratesa" "Günter" "Grass" "Una alegoría postnuclear narrada por una rata parlante que reflexiona sobre la historia" 2
crearFecha 08/10/2024
crearPrestamo
insertarLDEPrestamos

# Creo e inserto el Prestamo 3
crearSocio 34567890 Juan Diaz 10 03 2005
crearFecha 01/01/1968
crearLibro 431244557 "Lo bello y lo triste" "Yasunari" "Kawabata" "Un escritor se reencuentra con una amante del pasado en una historia de dolor y deseo" 1
crearFecha 08/11/2024
crearPrestamo
insertarLDEPrestamos

# Creo e inserto el Prestamo 4
crearSocio 45678901 Juan Diaz 10 03 1999
crearFecha 05/06/1967
crearLibro 843760494 "Cien años de soledad" "Gabriel" "García Márquez" "Es la historia de la familia Buendía a lo largo de siete generaciones en el pueblo ficticio de Macondo" 3
crearFecha 08/11/2025
crearPrestamo
insertarLDEPrestamos

obtenerPrimeroLDEPrestamos
obtenerUltimoLDEPrestamos


liberarLDEPrestamos
Fin
