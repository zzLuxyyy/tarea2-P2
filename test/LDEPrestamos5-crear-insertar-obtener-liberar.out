1>2>No se puede obtener el primero de la LDE de préstamos porque es vacía
3>No se puede obtener el último de la LDE de préstamos porque es vacía
4>#  Creo e inserto el Prestamo 1.
5>Socio de ci 12345678 creado exitosamente.
6>7>8>9>Préstamo de libro de isbn 411222335 a socio 12345678 creado exitosamente.
10>11>Ultimo préstamo:
Préstamo de libro Todos los fuegos el fuego a Jim Dear.
8/9/2024
No retornado
12>#  Creo e inserto el Prestamo 2.
13>Socio de ci 23456789 creado exitosamente.
14>15>16>17>Préstamo de libro de isbn 421233446 a socio 23456789 creado exitosamente.
18>19>#  Creo e inserto el Prestamo 3.
20>Socio de ci 34567890 creado exitosamente.
21>22>23>24>Préstamo de libro de isbn 431244557 a socio 34567890 creado exitosamente.
25>26>#  Creo e inserto el Prestamo 4.
27>Socio de ci 45678901 creado exitosamente.
28>29>30>31>Préstamo de libro de isbn 843760494 a socio 45678901 creado exitosamente.
32>33>Primer préstamo:
Préstamo de libro Todos los fuegos el fuego a Jim Dear.
8/9/2024
No retornado
34>Ultimo préstamo:
Préstamo de libro Cien años de soledad a Juan Diaz.
8/11/2025
No retornado
35>36>Fin.
